modules/op-resolver-git-config-scm.adoc

```diff
@@ -0,0 +1,66 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-config-scm_{context}"]
+= Configuring the Git resolver for the authenticated SCM API
+
+For the authenticated SCM API, you must set the configuration for the authenticatred Git connection.
+
+You can use Git repository providers that are supported by the link:https://github.com/jenkins-x/go-scm/tree/main/scm/driver[`go-scm` library]. Not all `go-scm` implementations have been tested with the Git resolver, but the following providers are known to work:
+
+* github.com and GitHub Enterprise
+* gitlab.com and self-hosted Gitlab
+* Gitea
+* BitBucket Server
+* BitBucket Cloud
+
+[NOTE]
+====
+You can configure only one Git connection using the authenticated SCM API for your cluster. This connection becomes available to all users of the cluster. All users of the cluster can access the repository using the security token that you configure for the connection.
+====
+
+[NOTE]
```

Comment:
We try to avoid back-to-back notes, especially of the same type. Can these be combined into one note (with bullets), or one of them moved elsewhere, or one of them brought out of being a note?